__tests__/fixtures/project/checkingResources/en_tit_checking/.apps/translationCore/checkData/selections/tit/1/4/2019-04-24T16_03_28.617Z.json

```diff
@@ -0,0 +1,28 @@
+{
+  "contextId": {
+    "reference": {
+      "bookId": "tit",
+      "chapter": 1,
+      "verse": 4
+    },
+    "tool": "translationWords",
+    "groupId": "godthefather",
+    "quote": "Θεοῦ Πατρὸς",
```

Comment:
why is this quote still a string?

Reply:
or was this done on purpose?

Reply:
Yes, that is the data before doing the resource updates.